src/lib/spinel/spinel.h

```diff
@@ -4731,6 +4731,24 @@ enum
      */
     SPINEL_PROP_RCP_ENH_ACK_PROBING = SPINEL_PROP_RCP_EXT__BEGIN + 3,
 
+    /// CSL Accuracy
+    /** Format: `c`
```

Comment:
Should it be capital `"C"`? (`"C"` is uint8 and `"c"` is int8).

Reply:
Right, should be `uint8_t`.